packages/devtools/devtools-testbench/src/App.tsx

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import React, { FC, useEffect, useState } from 'react';
+import { HashRouter } from 'react-router-dom';
+
+import { Client, fromHost } from '@dxos/client';
+import { ClientAndServices, PanelsContainer, sections } from '@dxos/devtools';
+import { AppView, OptionsContext, Routes as KaiRoutes } from '@dxos/kai';
+import { Config, Defaults } from '@dxos/config';
+import { ClientContext } from '@dxos/react-client';
+
+/**
+ * Main app container with routes.
+ */
+export const App: FC<{ views: AppView[]; debug?: boolean }> = ({ views, debug = false, demo = true }) => {
+  const [value, setValue] = useState<ClientAndServices | undefined>(undefined);
```

Comment:
```suggestion
  const [value, setValue] = useState<ClientAndServices>(undefined);
```
I believe these are equivalent